script/import_scripts/base.rb

```diff
@@ -630,6 +631,38 @@ def create_bookmarks(results, opts = {})
     [created, skipped]
   end
 
+  def create_likes(results, opts = {})
+    created = 0
+    skipped = 0
+    total = opts[:total] || results.count
+
+    created_by = User.new
+    post = Post.new
+
+    results.each do |result|
+      params = yield(result)
+
+      if params.nil?
+        skipped += 1
+      else
+        created_by = User.find_by(id: user_id_from_imported_user_id(params[:user_id]))
+        post = Post.find_by(id: post_id_from_imported_post_id(params[:post_id]))
```

Comment:
Anyways we could do this without doing 2 requests to the database for every like? Maybe we have an index in memory we can use to check the existence of the user and post?

Reply:
Hmm, let's optimize this when it starts to be an actual problem. Since bulk imports are the future, this seems like a premature and unnecessary optimization right now.